### PR TITLE
Implement inverse of git push --force

### PR DESCRIPTION
**Pull request by pieterdd**

### CODE REVIEW

File: gitalias.txt [hunk 1]
```diff
@@ -624,6 +624,9 @@
   # Get the current branch name
   branch-name = rev-parse --abbrev-ref HEAD
 
+  # Get name of the upstream branch
+  upstream-name = !git for-each-ref --format='%(upstream:short)' $(git symbolic-ref -q HEAD)
```
Comment (phdru) [inline gitalias.txt]: Nice trick!

Comment (joelparkerhenderson) [inline gitalias.txt]: Agree. That's a great quick win.

**Comment by pieterdd** [comment]
Yes. I originally chose a different name for the command and I forgot to change the reference to `upstream-name` there. My bad, I've updated it.

**Comment by joelparkerhenderson** [comment]
Ok good. In git, the "upstream" and "origin" can be different. I'll add the `reset-to-upstream` command now because it aligns with your `upstream-name` command. If you want something other than that, can you comment here?

This is a useful item:
http://stackoverflow.com/questions/2739376/definition-of-downstream-and-upstream/2749166#2749166

**Pull request merged**